pkg/cpi/commonUtils.go

```diff
@@ -148,3 +160,32 @@ func (httpFileDownloadRequestParameters HttpFileDownloadRequestParameters) Handl
 	log.Entry().Errorf("a HTTP error occurred! Response body: %v, Response status code : %v", responseBody, response.StatusCode)
 	return errors.Errorf("%s, Response Status code: %v", httpFileDownloadRequestParameters.ErrMessage, response.StatusCode)
 }
+
+// HandleHTTPFileUploadResponse - Handle the file upload response
+func (httpFileUploadRequestParameters HttpFileUploadRequestParameters) HandleHTTPFileUploadResponse() error {
+	response := httpFileUploadRequestParameters.Response
+	httpErr := httpFileUploadRequestParameters.HTTPErr
+	if response != nil && response.Body != nil {
+		defer response.Body.Close()
+	}
+
+	if response == nil {
+		return errors.Errorf("did not retrieve a HTTP response: %v", httpErr)
+	}
+
+	if response.StatusCode == http.StatusOK {
+		log.Entry().
+			WithField("Created Artifact", httpFileUploadRequestParameters.FilePath).
+			Info(httpFileUploadRequestParameters.SuccessMessage)
+		return nil
+	}
+	if httpErr != nil {
```

Comment:
CodeClimate says here:
>Similar blocks of code found in 2 locations. Consider refactoring.
    [cmd/integrationArtifactResource.go:223](https://github.com/SAP/jenkins-library/blob/master/cmd/integrationArtifactResource.go#L223)

Could you address this please.

Reply:
this change requires changes to already merged command, so ideally this will go as separate PR

Reply:
Ok, I agree that we address improvements to the DRYness in a separate PR